src/conversations/Conversations.ts

```diff
@@ -429,6 +438,67 @@ export default class Conversations {
     return seenPeers
   }
 
+  async newGroupConversation(
+    context: InvitationContext
```

Comment:
Are we sure we want to leave the responsibility of setting the metadata up to the caller? What if we made this method just take the list of members as an array and then constructed all the context for them? Feels less likely developers are going to mess that up

Reply:
Good call, fixed in https://github.com/xmtp/xmtp-js/pull/381/commits/6869df990910523f236b1863141e6f359f5f8dc5.